cocos/core/assets/asset.jsb.ts

```diff
@@ -35,9 +35,14 @@ import { createMap } from '../utils/js-typed';
 import { property } from '../data/class-decorator';
 import { getUrlWithUuid } from '../asset-manager/helper';
 import { extname } from '../utils/path';
-
+import { CCClass } from '../data/class';
+import { editorExtrasTag } from '../data/editor-extras-tag';
 declare const jsb: any;
 
+CCClass.fastDefine('jsb.CCObject', jsb.CCObject, { _name: '', _objFlags: 0, [editorExtrasTag]: {} });
```

Comment:
This piece of code should be sync to all portion that reference the `CCObject`. That's, you should define `CCObject.jsb.ts` and mapping it in `cc.config.json`.